features/facebookImport/test/utils/analyses-execution.js

```diff
@@ -17,10 +16,11 @@ export async function runAnalysisForExport(
     zipFile,
     pod = new MockPod()
 ) {
-    const facebookAccount = await importZip({
-        dataImporters,
+    const facebookAccount = new FacebookAccount();
+    await facebookAccount.import({
```

Comment:
Are you running this for the side effects? Because `import` should return something, right?

Reply:
Import just fills up the account as was before

Reply:
Er. Does it? It now *returns* a result, right? Or that's a different import?

Reply:
Yeah that's a different import. The account importes a zipFile with data and the importers import data from the dataset it's a little confusing

Reply:
Maybe make it a naming nit then?